sickbeard/subtitles.py

```diff
@@ -565,68 +798,93 @@ def dhm(td):
-                        logger.log(u'Subtitle search for {} {} delayed for {}'.format
-                                   (ep_to_sub['show_name'], episode_num(ep_to_sub['season'], ep_to_sub['episode']) or
-                                    episode_num(ep_to_sub['season'], ep_to_sub['episode'], numbering='absolute'),
-                                    dhm(lastsearched + delay_time - now)), logger.DEBUG)
+                        logger.log(u'Subtitle search for {0} {1} delayed for {2}'.format
+                                   (ep_to_sub['show_name'], ep_num, dhm(lastsearched + delay_time - now)), logger.DEBUG)
                         continue
 
                 show_object = Show.find(sickbeard.showList, int(ep_to_sub['showid']))
                 if not show_object:
-                    logger.log(u'Show with ID {} not found in the database'.format(ep_to_sub['showid']), logger.DEBUG)
+                    logger.log(u'Show with ID {0} not found in the database'.format(ep_to_sub['showid']), logger.DEBUG)
                     continue
 
                 episode_object = show_object.getEpisode(ep_to_sub['season'], ep_to_sub['episode'])
                 if isinstance(episode_object, str):
-                    logger.log(u'{} {} not found in the database'.format
-                               (ep_to_sub['show_name'], episode_num(ep_to_sub['season'], ep_to_sub['episode']) or
-                                episode_num(ep_to_sub['season'], ep_to_sub['episode'], numbering='absolute')), logger.DEBUG)
+                    logger.log(u'{0} {1} not found in the database'.format
+                               (ep_to_sub['show_name'], ep_num), logger.DEBUG)
                     continue
 
                 try:
                     episode_object.download_subtitles()
                 except Exception as error:
-                    logger.log(u'Unable to find subtitles for {} {}. Error: {}'.format
-                               (ep_to_sub['show_name'], episode_num(ep_to_sub['season'], ep_to_sub['episode']) or
-                                episode_num(ep_to_sub['season'], ep_to_sub['episode'], numbering='absolute'), ex(error)), logger.ERROR)
+                    logger.log(u'Unable to find subtitles for {0} {1}. Error: {2}'.format
+                               (ep_to_sub['show_name'], ep_num, ex(error)), logger.ERROR)
                     continue
 
             except Exception as error:
-                logger.log(u'Error while searching subtitles for {} {}. Error: {}'.format
-                           (ep_to_sub['show_name'], episode_num(ep_to_sub['season'], ep_to_sub['episode']) or
-                            episode_num(ep_to_sub['season'], ep_to_sub['episode'], numbering='absolute'), ex(error)), logger.ERROR)
+                logger.log(u'Error while searching subtitles for {0} {1}. Error: {2}'.format
+                           (ep_to_sub['show_name'], ep_num, ex(error)), logger.ERROR)
                 continue
```

Comment:
Sure... I'll do it later. The [old code](https://github.com/pymedusa/SickRage/pull/219/files?diff=split#diff-425be5582fb3c7b0fce3dbcd319fed75L595) was error as well